L1Trigger/Phase2L1ParticleFlow/README.md

```diff
@@ -0,0 +1,37 @@
+1) Basic Instructions
```

Comment:
if it's forbidden I can remove it

Reply:
It is an irrelevant private recipe. The official repository does not host those.

Reply:
Looks like what is an "irrelevant private recipe" and what is useful for CMS developers is a very personal opinion.

Is having it in the repository useful for the ongoing and future developments ?
(though it would probably need to be updated)

Reply:
The majority of this file constitutes outdated (and technically questionable) private recipes. Nobody using the official software needs or wants this information.

The single line instruction how to run the test config is fine.

Reply:
Well, no: **you** don't care about this recipe - which is fine - but you **don't know** who cares about it and why.

So, if you want to leave a comment like
```
This recipe looks out of date; is it still needed ? does it make sense to have it in CMSSW ?
```
it's perfectly fine.

On the other hand, a comment like
```
It is an irrelevant private recipe. The official repository does not host those.
```
is at the very least rude, possibly offensive, and - as I said - reflects only your personal opinion, not a fact.

Reply:
In the grand scheme of things, "rude" is trying to push O(100K) lines of code written with no attempt to follow the official code rules (or, in many cases, even basic best practices) at the last minute and then asking for a "quick" review and approval.

We cannot possibly maintain this repository if we're expected to care about what every developer may or may not have in their personal forks.

Reply:
> We cannot possibly maintain this repository if we're expected to care about what every developer may or may not have in their personal forks.

I don't think we've made such a terrible job over the pas 15 years ?

IMHO the point of a repository is to host the "code" that people will use - not just the workflow that the "central production" cares about.
And, as with everything, there are various degrees to what makes more or less sense to include...

As I already agreed: if the recipe is not useful, out of date, etc. - sure, let's drop it. But "forbidden" is a very strong word.